rootfs/builder/build.sh

```diff
@@ -199,8 +199,8 @@ if [[ "$slug_file" != "-" ]]; then
 					secretKey=`cat /var/run/secrets/object/store/access-secret-key`
 					domain=`echo $put_url | awk -F/ '{print $3}'`
 					echo $keyID $secretKey $domain
-					mc --quiet config host add $domain $keyID $secretKey
-					mc --quiet cp $slug_file $put_url/
+					/bin/mc --quiet config host add "http://$domain" $keyID $secretKey
```

Comment:
/bin is added to $PATH by default so you shouldn't need to give the absolute path here. :)